common-files/LICENSE.md

```diff
@@ -0,0 +1,10 @@
+The MIT License
+
+#set( $year = $package.getClass().forName("java.util.Date").newInstance().getYear() + 1900 )
```

Comment:
Why + 1900?

Reply:
embarrassingly copied that off stackoverflow for how to get the current year using velocity, see ref on why its offset by 1900, https://stackoverflow.com/a/9243597/4951015

deprecated since 1.1, replaced with clean localdate implementation